packages/fxa-content-server/app/scripts/models/auth_brokers/base.js

```diff
@@ -163,12 +167,18 @@ const BaseAuthenticationBroker = Backbone.Model.extend({
   /**
    * Request FXA_STATUS info from the UA.
    *
+   * @param {Object} [statusOptions] extra options for the status message.
    * @returns {Promise} resolves when complete.
    * @private
    */
-  _fetchFxaStatus () {
+  _fetchFxaStatus (statusOptions={}) {
     const channel = this._notificationChannel;
-    return channel.request(channel.COMMANDS.FXA_STATUS, this.relier.pick('service'))
+    const isPairing = statusOptions.isPairing;
+
+    return channel.request(channel.COMMANDS.FXA_STATUS, {
+      isPairing,
```

Comment:
@shane-tomlinson could you confirm this is okay? Blocking https://phabricator.services.mozilla.com/D29082

Reply:
Seems reasonable to me, mind adding some tests for it and I can r+?

Reply:
Thank you!